ocaml/idl/datamodel.ml

```diff
@@ -5587,6 +5590,33 @@ let storage_plugin =
      ])
     ()
 
+let lvhd_enable_thin_provisioning = call
+   ~name:"enable_thin_provisioning"
+   ~in_oss_since:None
+   ~in_product_since:rel_dundee
+   ~allowed_roles:_R_POOL_ADMIN
+   ~params:[
+     Ref _sr, "SR", "The LVHD SR to upgrade to being thin-provisioned.";
+     Int, "initial_allocation", "The initial amount of space to allocate to a newly-created VDI in bytes";
```

Comment:
The original conversation was about how SM and XC agreed on passing info using the sm-config field. That field is custom (in fact we are passing fractions now) so, as long it is clear
to the user, we can decide any kind of arrangement. Given the minimum granularity for that field is above MBs, that is the reasonable choice.

I do not know how this can break any xapi model, given this field did not exist and it is a custom SM field..